README.md

```diff
@@ -1,2 +1,80 @@
-# ai_on_edge_with_fiware
-This is a project to implement an AI on Edge application with FIWARE. 
+# Context Broker environment setup
+
+This branch contains the setup for running an Orion Context Broker connected to a MongoDB instance using Docker Compose. The architecture is designed to facilitate easy deployment and management of the Context Broker and its database.
+
+## Architecture
+![Architecture Diagram](docs/architecture.png) 
+
+The architecture consists of two main components:
+- [**Orion-LD Context Broker**](https://fiware-orion.readthedocs.io/en/latest/): Acts as the central component for managing context information.
+- [**MongoDB**](https://www.mongodb.com/): Serves as the database for storing context data.
+
+The two components are connected, with Orion-LD Context Broker exposed on port 1026.
+
+## Prerequisites
+
+To keep things simple, all components will be run using Docker. Docker is a container technology that allows different components to be isolated into their respective environments.
+
+### Installing Docker
+
+- **Windows**: Follow the instructions [here](https://docs.docker.com/desktop/install/windows-install/).
+- **Mac**: Follow the instructions [here](https://docs.docker.com/desktop/install/mac-install/).
+- **Linux**: Follow the instructions [here](https://docs.docker.com/desktop/install/linux-install/).
+
+### Installing Docker Compose
+
+Docker Compose is a tool for defining and running multi-container Docker applications. A series of YAML files are used to configure the required services for the application. This means all container services can be brought up in a single command.
+
+- Docker Compose is installed by default as part of Docker for Windows and Docker for Mac.
+- **Linux**: Follow the instructions [here](https://docs.docker.com/compose/install/).
+
+### Checking Docker and Docker Compose Versions
+
+You can check your current Docker and Docker Compose versions using the following commands:
+
+```sh
+docker-compose -v
+docker version
+```
```

Comment:
I think this is not necessary, we have to add a prerequesite part on top and say, make sure to have docker installed, docker-compose, etc.. and we add to it things throughout the way.

Reply:
in a bullet point manner, we do not care how they install it! because the documentation should only provide guidelines on our use case only. e.g how to run the container, that would be a useful command.